container_definitions/amd64/connectome-manipulator/spack.yaml

```diff
@@ -1,12 +1,13 @@
+---
 spack:
-  specs:
-    - parquet-converters
-    - py-connectome-manipulator
+  specs: [parquet-converters, py-connectome-manipulator]
+  concretizer:
+    unify: when_possible
```

Comment:
uh what is going on here? what was the problem?

Reply:
It's part of the workaround for #10 ; seems like spack was having trouble with our `concretizer:unify:true` setting because different packages needed different versions of the same dependency.

I loosened the unification a little bit for this particular container to allow multiple versions of the same package in the concretization.